src/styles/blocks/rate.scss

```diff
@@ -0,0 +1,41 @@
+.rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__text {
+    height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $darcBlue;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    padding: 0;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+
+  &__stars {
+    display: flex;
+    margin: 0;
+    padding: 0;
+
+    &--1 .rate__star:first-child,
+    &--2 .rate__star:nth-child(-n + 2),
+    &--3 .rate__star:nth-child(-n + 3),
+    &--4 .rate__star:nth-child(-n + 4),
+    &--5 .rate__star:nth-child(-n + 5) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
You can rewrite it using @for loop

Reply:
yes, use `for` loop to generate these selectors